runtime/autoload/tutor.vim

```diff
@@ -258,9 +258,9 @@ endfunction
 
 function! s:GlobPath(lp, pat)
     if version >= 704 && has('patch279')
-        return globpath(a:lp, a:pat, 0, 1)
+        return globpath(a:lp, a:pat, 1, 1)
```

Comment:
If user installs runtime to a path that matches `'wildignore'`, this will fail. In general, always set this argument.